src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -20,78 +20,155 @@
+ *      1       10108   .       CAACCCT C       46514.32        PASS    AC=6;AF=1.525E-4
+ *      1       10109   .       AACCCTAACCCT    AAACCCT,*       89837.27        PASS    AC=48,5;AF=0.001223,1.273E-4
+ *      1       10114   .       TAACCCTAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCTAACCCTAACCCTAACCCTAACCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCTAACCCTAAACCCTA  *,CAACCCTAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCTAACCCTAACCCTAACCCTAACCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCTAACCCTAAACCCTA,T      36728.97        PASS    AC=55,9,1;AF=0.001373,2.246E-4,2.496E-5
+ *      1       10119   .       CT      C,*     251.23  PASS    AC=5,1;AF=1.249E-4,2.498E-5
+ *      1       10120   .       TA      CA,*    14928.74        PASS    AC=10,6;AF=2.5E-4,1.5E-4
+ *      1       10128   .       ACCCTAACCCTAACCCTAAC    A,*     285.71  PASS    AC=3,1;AF=7.58E-5,2.527E-5
+ *      1       10131   .       CT      C,*     378.93  PASS    AC=7,5;AF=1.765E-4,1.261E-4
+ *      1       10132   .       TAACCC  *,T     18025.11        PASS    AC=12,2;AF=3.03E-4,5.049E-5
+ * </pre>
+ *
+ * <h3>How Mutect2 works compared to HaplotypeCaller</h3>
+ * <p>Overall, Mutect2 works similarly to HaplotypeCaller, but with a few key differences. </p>
+ * <p>
+ *     (i) GVCF calling is not a feature of Mutect2.
+ *     (ii) While HaplotypeCaller relies on a fixed ploidy assumption to inform its genotype likelihoods that are the basis for genotype probabilities (PL),
+ *     Mutect2 allows for varying ploidy in the form of allele fractions for each variant.
+ *     Varying allele fractions is often seen within a tumor sample due to fractional purity, multiple subclones and/or copy number variation.
+ *     (iii) Mutect2 also differs from the HaplotypeCaller in that it can apply various prefilters to sites and variants depending on the use of
+ *     a matched normal (--normalSampleName), a panel of normals (PoN; --normal_panel) and/or a common population variant resource containing allele-specific frequencies (--germline_resource).
+ *     If provided, Mutect2 uses the PoN to filter sites and the germline resource and matched normal to filter alleles.
+ *     (iv) Mutect2 variant site annotations differ from that of HaplotypeCaller. See the --annotation parameter description for a list.
+ *     (v) Finally, Mutect2 has additional parameters not available to HaplotypeCaller that factor in the decision to reassemble a genomic region,
+ *     factor in likelihood calculations that then determine whether to emit a variant, or factor towards filtering.
+ *     These parameters include the following and are each described further in the arguments section.
+ * </p>
+ *
+ * <dl>
+ *     <dd>--min_variants_in_pileup ==> active region determination</dd>
+ *     <dd>--minNormalVariantFraction ==> active region determination</dd>
+ *     <dd>--tumorStandardDeviationsThreshold ==> active region determination</dd>
```

Comment:
Just FYI, we realized recently that M2's active region determination, while better than GATK3, still has a very long way to go.  We're going to blow up many of these parameters soon.